domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -15,13 +15,26 @@
 
     private static final String AND_CONDITION_REGEX = "\\sAND\\s(?=(([^\"]*\"){2})*[^\"]*$)";
     private static final String AND_OPERATOR = " AND ";
+    private static final String OR_CONDITION_REGEX = "\\sOR\\s(?=(([^\"]*\"){2})*[^\"]*$)";
+    private Pattern orConditionPattern = Pattern.compile(OR_CONDITION_REGEX);
+    private static final String OR_OPERATOR = " OR ";
     private static final Pattern EQUALITY_CONDITION_PATTERN = Pattern.compile("\\s*?(.*)\\s*?(=|CONTAINS)\\s*?(\".*\")\\s*?");
+    private static final Pattern NOT_EQUAL_CONDITION_PATTERN = Pattern.compile("\\s*?(.*)\\s*?(!=|CONTAINS)\\s*?(\"" +
+                                                                               ".*\")\\s*?");
 
     public ShowCondition parseShowCondition(String rawShowConditionString) throws InvalidShowConditionException {
         try {
             if (rawShowConditionString != null) {
-                String[] andConditions = rawShowConditionString.split(AND_CONDITION_REGEX);
-                Optional<ShowCondition> optShowCondition = buildShowCondition(andConditions);
+                String andOrOperator = AND_OPERATOR;
```

Comment:
We are changing the logic, but I cannot see any tests updated, added in ShowConditionParserTest.

Reply:
Added, DONE